### PR TITLE
33511: Updated phpmd/phpmd composer dependency to version 2.9.1

### PR DESCRIPTION
**Pull request by anzin**
### Description (*)

I've updated phpmd/phpmd composer dependency to version 2.9.1

### Fixed Issues (if relevant)

1. Fixes magento/magento2#33511

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run WebAPI Tests

**Review by Den4ik — APPROVED** [review]

File: composer.lock [hunk 1]
```diff
@@ -11304,5 +11315,5 @@
         "lib-libxml": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "2.1.0"
```
Comment (eduard13) [inline composer.lock]: @anzin could you please generate it using 2.0.0 composer version?

Comment (anzin) [reply composer.lock]: Hello @eduard13, thanks for your review, I corrected version 'plugin-api-version' to 2.0.0

**Review by Den4ik — APPROVED** [review]
✅ Approved.
Thanks for contribution

**Review by eduard13 — APPROVED** [review]

**Comment by eduard13** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run WebAPI Tests, Functional Tests B2B

**Review by eduard13 — APPROVED** [review]

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Review by andrewbess — APPROVED** [review]
Hello @anzin 
Your changes look good for me.
Thank you for your job

**Review by Den4ik — APPROVED** [review]
✅ Approved.
@anzin Thanks for work

**Pull request merged**